code/modules/surgery/organs/pain.dm

```diff
@@ -1,62 +1,47 @@
-mob/var/list/pain_stored = list()
-mob/var/last_pain_message = ""
```

Comment:
Yes, Bay, what a great idea, let's store pain messages on ghost mobs!

Reply:
eternal languish and fury of the damned salty souls